lib/get-message.js

```diff
@@ -1,11 +1,30 @@
-const _ = require('lodash')
+const { defaultCommitMessages } = require('./default-commit-messages')
 
 module.exports = function (commitMessages, key, values) {
-  if (!Object.prototype.hasOwnProperty.call(commitMessages, key)) {
+  if (!Object.prototype.hasOwnProperty.call(commitMessages, key) &&
+  !Object.prototype.hasOwnProperty.call(defaultCommitMessages, key)) {
     throw new Error(`Unknown message key '${key}'`)
   }
 
   const templateValues = Object.assign({}, values)
 
-  return _.template(commitMessages[key])(templateValues)
+  function replaceMessageVariables (message, variables) {
```

Comment:
`variables` here is always assigned to `templateValues`.

Also it might be nice to use `_.reduce` in here instead of working with `Object.keys(..).forEach)`

Reply:
Also, assuming the `templateValues` issue is fixed it's likely this function can be moved out of the main function and being redeclared on every call to `get-message`.

Reply:
I failed to rewrite the function with reduce 🙈 .. I gave up.